core/src/components/badge/badge.tsx

```diff
@@ -47,6 +47,14 @@ export class Badge implements ComponentInterface {
    */
   @Prop() size?: 'xxsmall' | 'xsmall' | 'small' | 'medium' | 'large' | 'xlarge';
 
+  /**
+   * Set to `"top-right"` to position the badge on top right absolute position of the parent element.
+   * Set to `"bottom-right"` to position the badge on bottom right absolute position of the parent element.
+   *
+   * Defaults to `"static"`.
+   */
+  @Prop() position: 'top-right' | 'bottom-right' | 'static' = 'static';
```

Comment:
```suggestion
  @Prop() position?: 'top-right' | 'bottom-right';
```

Can we treat `undefined` as the static option?

Reply:
I think so, I added the static to create a selector to easily make CSS styles, in this case we just add position: absolute using :not(.position-static). But I can make a more complex :not without this, will make the change.